tests/Makefile

```diff
@@ -11,16 +11,23 @@ default: it
 
 TESTBINS = unittest_stralloc unittest_blast unittest_prioq
 
+SINGLETESTBIN = unittest_prioq
```

Comment:
not sure if there's a way that I can run just a single test case and apparently, one option that I can see right now is to add target to run just a single specific test binary, it's better than run all of the test binary, but it still run all of the test cases in a single test binary

Reply:
`$ make test TESTBINS="unittest_stralloc unittest_prioq"`

(and `TESTBINS` can of course have just one value, too)